pyanaconda/ui/gui/spokes/network.py

```diff
@@ -429,6 +436,25 @@ def add_connection_to_list(self, con):
             self.add_dev_cfg(dev_cfg)
         return True
 
+    def _get_vlan_interface_name_from_connection(self, connection):
+        """Get vlan interface name from vlan connection.
+
+        If no interface name is specified in the connection settings, infer
+        the value as <PARENT_IFACE>.<VLAN_ID> - same as NetworkManager.
+        """
+        iface = connection.get_setting_connection().get_interface_name()
+        if not iface:
+            setting_vlan = connection.get_setting_vlan()
+            if setting_vlan:
+                vlanid = setting_vlan.get_id()
+                parent = setting_vlan.get_parent()
+                # if parent is specified by UUID
+                if len(parent) == 36:
```

Comment:
What if the parent will have name with 36 characters but it won't be UUID ? Could that happen?

Reply:
No, it can be either UUID or network iface name and the name size is limited to 15 or 16 chars in kernel

Reply:
ok, then it should be fine :)